chainerui/models/result.py

```diff
@@ -43,17 +45,59 @@ def __init__(self, path_name=None, name=None, project_id=None,
     def __repr__(self):
         return '<Result id: %r, path_name: %r />' % (self.id, self.path_name)
 
-    @property
-    def serialize(self):
-        """serialize."""
+    @classmethod
+    def create(cls, path_name=None, name=None, project_id=None,
+               log_modified_at=None):
+        """Initialize an instance and save it to db."""
+        result = cls(path_name, name, project_id, log_modified_at)
+
+        DB_SESSION.add(result)
+        DB_SESSION.commit()
+
+        crawl_result.crawl_result(result.id, True)
```

Comment:
I think this line should be `crawl_result(result.id, True)` , but

- `crawl_result(result.id, True)` fails in python3
- `crawl_result.crawl_result(result.id, True)` works in python3
- `crawl_result.crawl_result(result.id, True)` fails in python2

I couldn't figure out why.